src/main/java/com/google/sps/servlets/PopulateUserEntity.java

```diff
@@ -32,15 +32,26 @@
 import com.google.gson.Gson;
 
 @WebServlet("/user")
```

Comment:
Please use a more specific path than "/user" -- I expect you'll have more than one user-related servlet.